JoonasHeinonen/escape-from-zordoom-prison
# Features/godot4 menu and arena

## [godot4 menu and arena / description](https://trello.com/b/vThoQnyu/escape-from-zordoom-prison)


## Changes:
Upgraded from godot 3 to 4.2
fixed the menu buttons
fixed the images and text to work in the character selection screen
Refactor nef head enemy where it can find a cliff and can turn around to shoot at the player.
Got Dialogic 2 to work with shark man so that the player can read through the text and once it is done select what fight to do.


## How to test:
 Pull down the branch
go to the arena level
make sure you can go to the shark man npc and talk to him
once done talking to him you can interact with the arena menu
select the first fight 
fight the first nef head and make sure he can follow the player to the end of the level to turn around
kill the nef head and head back to the shark man npc with 100 bolts.

## Sources cited:
